src/Lecture1.hs

```diff
@@ -63,7 +70,11 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+toPositive :: Int -> Int
+
+toPositive n = if (n > 0) then n else negate n
```

Comment:
That's correct! 💯 
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```